### ISSUE TITLE
req.context.invocationId

### ISSUE THREAD

**Issue by espray** [yvele/azure-function-express#6]
Can the invocationId made available through req.context?

**Comment by yvele** [comment]
@espray I think I should make the whole `context` object available.. What do you think?

**Comment by espray** [comment]
I thought that at first, but that might be problematic.
If `context.done()` was called it would mess with the express flow.
`context.bindings` are mapped to the `express.Request` and `express.Response` params.

The safe play is `log` and `invocationId` only

Also, here is my Typescript declaration file `azure-function-express.d.ts`
```
import * as express from 'express';

export interface AzureFunctionExpressRequest extends express.Request {
  context: {
    invocationId: string;
    log(message: any): void;
  }
}
```

**Comment by yvele** [comment]
@espray I don't like whitelisting and sometimes prefer giving responsibilities to developers but in this case yep I will explicitly add `invocationId`

But hum.. I can't find `context.contextId` in any [Azure documentation](https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node#context-object) or via a [Google search](https://www.google.fr/search?q=azure+function+context.invocationId). Can you point me to the doc?

**Comment by espray** [comment]
Yup I agree on giving full responsibilities to developers.

`context.invocationId'
https://github.com/christopheranderson/azure-functions-typescript
You could ask Christopher, his git profile says he is a Microsoft PM - Azure.
- OR - 
`context.bindingData.InvocationId`
https://github.com/Azure/azure-webjobs-sdk-script/issues/504

**Comment by yvele** [comment]
@espray I've published a new patch [v1.2.2](https://github.com/yvele/azure-function-express/releases/tag/v1.2.2) to NPM ;)

**Comment by yvele** [comment]
@espray I've published a new patch [v1.2.2](https://github.com/yvele/azure-function-express/releases/tag/v1.2.2) to NPM ;)
